source/collections/CircularQueue.ooc

```diff
@@ -8,10 +8,11 @@
 
 import VectorQueue
 
+// Dequeues the buffer when maximum capacity is reached
 CircularQueue: class <T> extends Queue<T> {
 	_backend: VectorQueue<T>
 	_cleanupCallback: Func (T)
-	_maximumCapacity: Int
+	maximumCapacity: Int
```

Comment:
There already is a `capacity` which returns the backend capacity, equal to `this _maximumCapacity`.